### PR TITLE
Refactor remaining create, delete, export, get cmd to use adapter

### PR DESCRIPTION
**Pull request by SomtochiAma**
Part of: #291 

Signed-off-by: Somtochi Onyekwere <somtochionyekwere@gmail.com>

### CODE REVIEW

**Review by stefanprodan — APPROVED** [review]
LGTM

Thanks @SomtochiAma

**Review by hiddeco — COMMENTED** [review]
Copyright headers are missing on newly added files.

File: cmd/flux/export.go [hunk 1]
```diff
@@ -110,6 +108,74 @@ func (export exportCommand) run(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+//func (export exportWithSecretCommand) run(cmd *cobra.Command, args []string) error {
```
Comment (stefanprodan) [inline cmd/flux/export.go:111]: Delete all this code please

**Review by stefanprodan — CHANGES_REQUESTED** [review]

**Review by hiddeco — APPROVED** [review]
Needs a :octopus: (squash), other than that LGTM :100:

**Pull request merged**